src/topology/algebra/monoid.lean

```diff
@@ -101,6 +101,24 @@ lemma filter.tendsto.mul_const (b : M) {c : M} {f : α → M} {l : filter α}
   (h : tendsto (λ (k:α), f k) l (𝓝 c)) : tendsto (λ (k:α), f k * b) l (𝓝 (c * b)) :=
 h.mul tendsto_const_nhds
 
+lemma filter.tendsto_nhds_within_pos.const_mul {α 𝕜 : Type*} {l : filter α}
```

Comment:
```suggestion
lemma filter.tendsto_nhds_within_Ioi.const_mul {α 𝕜 : Type*} {l : filter α}
```
I think `nhds_within_pos` is only used for `𝓝[>] 0`, and that `nhds_within_Ioi` is much more frequent in mathlib. Same thing below.

Reply:
Done.